.circleci/config.yml

```diff
@@ -110,6 +110,7 @@ jobs:
     executor: golang
     steps:
     - checkout
+    - run: curl -sL https://github.com/mikefarah/yq/releases/download/v4.6.3/yq_linux_amd64 -O "${PATH%%:*}/yq" && chmod -v +x "${PATH%%:*}/yq"
```

Comment:
can we hardcode the checksum and verify it? this script has access to our secret

Reply:
Done.